tests/integration_tests/common_1/run.sh

```
@@ -65,6 +65,8 @@ EOF
 
 	# sync_diff can't check non-exist table, so we check expected tables are created in downstream first
 	check_table_exists common_1.v1 ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
+	check_table_exists common_1.users ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)